components/org.wso2.carbon.consent.mgt.endpoint/src/gen/java/org/wso2/carbon/consent/mgt/endpoint/dto/ConsentResponseDTO.java

```diff
@@ -90,6 +96,30 @@ public void setState(String state) {
   }
 
   
+  /**
+   **/
+  @ApiModelProperty(value = "")
+  @JsonProperty("spDisplayName")
+  public String getSpDisplayName() {
+    return spDisplayName;
+  }
```

Comment:
add new line after

Reply:
These are auto geneated codes; swagger to cxf